src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+
+/// создание head
+const createTh = document.createElement('th');
+
+createTh.textContent = table.querySelectorAll('thead tr th')[1].textContent;
+
+const vst = table.querySelectorAll('thead tr th')[3];
```

Comment:
Ensure that the table has at least four columns in the header. Accessing `table.querySelectorAll('thead tr th')[3]` will throw an error if there are fewer than four columns.